modules/prefixer.js

```diff
@@ -385,13 +386,16 @@ var getPrefixedStyle = function (style, mode /* 'css' or 'js' */) {
       // Ignore unsupported properties
       /* eslint-disable no-console */
       if (console && console.warn) {
-        console.warn('Unsupported CSS property "' + property + '"');
+        console.warn(
+          'Unsupported CSS property "' + property +
+          '" in component "' + component.constructor.displayName + '"'
```

Comment:
Ah yes, that wouldn't be fun :) I will add a check to see whether we have a component or not.